llvm/docs/LangRef.rst

```diff
@@ -16574,21 +16574,35 @@ type.
 
 Semantics:
 """"""""""
+Follows the IEEE-754 semantics for minNum, except that -0.0 < +0.0 for the purposes
+of this intrinsic. As for signaling NaNs, per the IEEE-754 semantics, if either operand
```

Comment:
This is the IEEE-754 behavior (just the 2019 version)

Reply:
The names of the operations are different in 2008 and 2019.
```   
2008 minNum
2019 minimumNumber
```

I don't think that it is a good idea to mention 2019 here, which will confuse the reader.
I will change IEEE-754 to IEEE754-2008 then to make it more clear.